specs/electra/beacon-chain.md

```diff
@@ -30,35 +34,37 @@
 
 ## Introduction
 
-This is the beacon chain specification of in-protocol deposits processing mechanism.
-This mechanism relies on the changes proposed by [EIP-6110](http://eips.ethereum.org/EIPS/eip-6110).
-
-*Note:* This specification is built upon [Deneb](../../deneb/beacon-chain.md) and is under active development.
+Electra is a consensus-layer upgrade containing a number of features. Including:
+* [EIP-6110](https://eips.ethereum.org/EIPS/eip-6110): Supply validator deposits on chain
+* [EIP-7002](https://eips.ethereum.org/EIPS/eip-7002): Execution layer triggerable exits
 
 ## Constants
 
 The following values are (non-configurable) constants used throughout the specification.
 
 ### Misc
 
-| Name | Value |
-| - | - |
-| `UNSET_DEPOSIT_RECEIPTS_START_INDEX` | `uint64(2**64 - 1)` |
+| Name | Value | Description |
+| - | - | - |
+| `UNSET_DEPOSIT_RECEIPTS_START_INDEX` | `uint64(2**64 - 1)` | *[New in Electra:EIP6110]* |
 
 ## Preset
 
 ### Execution
 
 | Name | Value | Description |
 | - | - | - |
-| `MAX_DEPOSIT_RECEIPTS_PER_PAYLOAD` | `uint64(2**13)` (= 8,192) | Maximum number of deposit receipts allowed in each payload |
+| `MAX_DEPOSIT_RECEIPTS_PER_PAYLOAD` | `uint64(2**13)` (= 8,192) | *[New in Electra:EIP6110]* Maximum number of deposit receipts allowed in each payload |
+| `MAX_EXECUTION_LAYER_EXITS` | `2**4` (= 16) |  *[New in Electra:EIP7002]* |
 
 ## Containers
 
 ### New containers
 
 #### `DepositReceipt`
 
+*Note*: The container is new in EIP6110.
```

Comment:
This note can be inferred from 
```
 deposit_receipts: List[DepositReceipt, MAX_DEPOSIT_RECEIPTS_PER_PAYLOAD]  # [New in Electra:EIP6110]`
```
not sure if it's worth the percent and now we may have to annotate all containers